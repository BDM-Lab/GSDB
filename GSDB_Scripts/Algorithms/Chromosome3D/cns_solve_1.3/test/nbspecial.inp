{====>}
evaluate ($site_type="YB") { Specify type of sites. }


topology residue site atom site mass=10.   charge=0.  type=$site_type end end end
segment name = " " molecule number=4 name=site end end
do (x=0.)  ( all ) !
do (y=0.)  ( all ) !
do (z=0.)  ( all ) ! everything needs to be initialized!
do (q=1.0) ( all ) !
do (b=20.) ( all ) !

coor
ATOM      1 SITE SITE    1      13.934   0.000  15.359  1.00 20.00          
ATOM      2 SITE SITE    2      11.744  10.360   9.452  1.00 20.00          
ATOM      3 SITE SITE    3      -2.635   6.475  55.530  1.00 20.00          
ATOM      4 SITE SITE    4      -3.322   7.770  54.348  1.00 20.00          
end

do (x=0.)  ( resid 1 ) !
do (y=0.)  ( resid 1 ) !
do (z=0.)  ( resid 1 ) ! everything needs to be initialized!

{====>}                    { Space group. Uses International Table conventions.  } 
                           { with subscripts substituted by parenthesis.         }                                            
evaluate ($SG="C2" ) 

{====>}                                                             { Unit cell. }
 evaluate ($a=97.38)
 evaluate ($b=46.62)
 evaluate ($c=65.39)
 evaluate ($alpha=90.0)
 evaluate ($beta=115.39)
 evaluate ($gamma=90.0)



xray                                    

  symmetry=( X,Y,Z )
  symmetry=( -X,Y,-Z )
  symmetry=( 1/2+X,1/2+Y,Z )
  symmetry=( 1/2-X,1/2+Y,-Z )

   a=$a b=$b c=$c  alpha=$alpha beta=$beta gamma=$gamma      { Define unit cell. }

end





parameter 
   evaluate ($max_dist=2*max($a,$b,$c))
   evaluate ($max_dist_on=$max_dist-1.0)
   evaluate ($max_dist_off=$max_dist-0.5)
   nbonds cutnb=$max_dist wmin=0.0 repel=1.0 rexp=4 irex=1 rcons=16 tole=0.5 nbxmod=5
        ctonnb=$max_dist_on ctofnb=$max_dist_off 
        special=-1
   end
   evaluate ($vdw_radius=2)
   evaluate ($vdw_radius   = $vdw_radius / 2^(1/6))
   nonbonded  $site_type   0.1  $vdw_radius    0.1 $vdw_radius  
end

flags excl * incl vdw pvdw end

eval ($current_site=4) 

igroup interaction=( resid 1:$current_site) ( resid 1:$current_site ) end
distance from=( resid 1:$current_site ) to=( resid 1:$current_site) cuton=0.0 cutoff=$max_dist 
         disposition=rmsd
end

show elem ( rmsd ) ( resid 1:$current_site ) 

xray
   scatter ( all ) 0 0 0 0 0 0 0 0 1
   special
      selection=( all ) 
      to=rmsd
   end
end

show elem ( rmsd ) ( all ) 

      

stop


