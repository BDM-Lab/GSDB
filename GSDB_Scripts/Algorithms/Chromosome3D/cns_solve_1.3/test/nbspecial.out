          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:04 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>{====>} 
 CNSsolve>evaluate ($site_type="YB") { Specify type of sites. } 
 EVALUATE: symbol $SITE_TYPE set to "YB" (string)
 CNSsolve> 
 CNSsolve> 
 CNSsolve>topology residue site atom site mass=10.   charge=0.  type=$site_type end end end 
 CNSsolve>segment name = " " molecule number=4 name=site end end 
 Status of internal molecular topology database:
 -> NATOM=          4(MAXA=     1000000)  NBOND=          0(MAXB=     1000000)
 -> NTHETA=         0(MAXT=     2000000)  NGRP=           4(MAXGRP=   1000000)
 -> NPHI=           0(MAXP=     2000000)  NIMPHI=         0(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve>do (x=0.)  ( all ) ! 
 SELRPN:      4 atoms have been selected out of      4
 CNSsolve>do (y=0.)  ( all ) ! 
 SELRPN:      4 atoms have been selected out of      4
 CNSsolve>do (z=0.)  ( all ) ! everything needs to be initialized! 
 SELRPN:      4 atoms have been selected out of      4
 CNSsolve>do (q=1.0) ( all ) ! 
 SELRPN:      4 atoms have been selected out of      4
 CNSsolve>do (b=20.) ( all ) ! 
 SELRPN:      4 atoms have been selected out of      4
 CNSsolve> 
 CNSsolve>coor 
 COOR>ATOM      1 SITE SITE    1      13.934   0.000  15.359  1.00 20.00 
 COOR>ATOM      2 SITE SITE    2      11.744  10.360   9.452  1.00 20.00 
 %READC-WRN: multiple coordinates for      4 atoms
 CNSsolve> 
 CNSsolve>do (x=0.)  ( resid 1 ) ! 
 SELRPN:      1 atoms have been selected out of      4
 CNSsolve>do (y=0.)  ( resid 1 ) ! 
 SELRPN:      1 atoms have been selected out of      4
 CNSsolve>do (z=0.)  ( resid 1 ) ! everything needs to be initialized! 
 SELRPN:      1 atoms have been selected out of      4
 CNSsolve> 
 CNSsolve>{====>}                    { Space group. Uses International Table conventions.  } 
 CNSsolve>                           { with subscripts substituted by parenthesis.         } 
 CNSsolve>evaluate ($SG="C2" ) 
 EVALUATE: symbol $SG set to "C2" (string)
 CNSsolve> 
 CNSsolve>{====>}                                                             { Unit cell. } 
 CNSsolve> evaluate ($a=97.38) 
 EVALUATE: symbol $A set to    97.3800     (real)
 CNSsolve> evaluate ($b=46.62) 
 EVALUATE: symbol $B set to    46.6200     (real)
 CNSsolve> evaluate ($c=65.39) 
 EVALUATE: symbol $C set to    65.3900     (real)
 CNSsolve> evaluate ($alpha=90.0) 
 EVALUATE: symbol $ALPHA set to    90.0000     (real)
 CNSsolve> evaluate ($beta=115.39) 
 EVALUATE: symbol $BETA set to    115.390     (real)
 CNSsolve> evaluate ($gamma=90.0) 
 EVALUATE: symbol $GAMMA set to    90.0000     (real)
 CNSsolve> 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>xray 
 XRAY> 
 XRAY>  symmetry=( X,Y,Z ) 
 XRAY>  symmetry=( -X,Y,-Z ) 
 XRAY>  symmetry=( 1/2+X,1/2+Y,Z ) 
 XRAY>  symmetry=( 1/2-X,1/2+Y,-Z ) 
 XRAY> 
 XRAY>   a=$a b=$b c=$c  alpha=$alpha beta=$beta gamma=$gamma      { Define unit cell. } 
 XRAY> 
 XRAY>end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve> 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR>   evaluate ($max_dist=2*max($a,$b,$c)) 
 EVALUATE: symbol $MAX_DIST set to    194.760     (real)
 PARRDR>   evaluate ($max_dist_on=$max_dist-1.0) 
 EVALUATE: symbol $MAX_DIST_ON set to    193.760     (real)
 PARRDR>   evaluate ($max_dist_off=$max_dist-0.5) 
 EVALUATE: symbol $MAX_DIST_OFF set to    194.260     (real)
 PARRDR>   nbonds cutnb=$max_dist wmin=0.0 repel=1.0 rexp=4 irex=1 rcons=16 tole=0.5 nbxmod=5 
 NBDSET>        ctonnb=$max_dist_on ctofnb=$max_dist_off 
 NBDSET>        special=-1 
 NBDSET>   end 
 PARRDR>   evaluate ($vdw_radius=2) 
 EVALUATE: symbol $VDW_RADIUS set to    2.00000     (real)
 PARRDR>   evaluate ($vdw_radius   = $vdw_radius / 2^(1/6)) 
 EVALUATE: symbol $VDW_RADIUS set to    1.78180     (real)
 PARRDR>   nonbonded  $site_type   0.1  $vdw_radius    0.1 $vdw_radius 
 PARRDR>end 
 CNSsolve> 
 CNSsolve>flags excl * incl vdw pvdw end 
 CNSsolve> 
 CNSsolve>eval ($current_site=4) 
 EVALUATE: symbol $CURRENT_SITE set to    4.00000     (real)
 CNSsolve> 
 CNSsolve>igroup interaction=( resid 1:$current_site) ( resid 1:$current_site ) end 
 SELRPN:      4 atoms have been selected out of      4
 SELRPN:      4 atoms have been selected out of      4
 CNSsolve>distance from=( resid 1:$current_site ) to=( resid 1:$current_site) cuton=0.0 cutoff=$max_dist 
 SELRPN:      4 atoms have been selected out of      4
 SELRPN:      4 atoms have been selected out of      4
 DISTANCE>         disposition=rmsd 
 DISTANCE>end 
 DISTAN: minimum nonbonded distances copied to RMSD array
 NBONDS: generating intra-molecular exclusion list with mode= 5
 MAKINB: Ran out of space. RESIZING
 MAKINB: mode   5 found      0 exclusions and      0 interactions(1-4)
 NBONDS: found        2 intra-atom interactions
 NBONDS: found       34 symmetry-atom interactions
 CNSsolve> 
 CNSsolve>show elem ( rmsd ) ( resid 1:$current_site ) 
 SELRPN:      4 atoms have been selected out of      4
 (      SITE 1    SITE )    0.0000    
 (      SITE 2    SITE )    18.292    
 (      SITE 3    SITE )    1.8831    
 (      SITE 4    SITE )    1.8831    
 CNSsolve> 
 CNSsolve>xray 
 XRAY>   scatter ( all ) 0 0 0 0 0 0 0 0 1 
 SELRPN:      4 atoms have been selected out of      4
 XRAY>   special 
 SPEcial>      selection=( all ) 
 SELRPN:      4 atoms have been selected out of      4
 SPEcial>      to=rmsd 
 SPEcial>   end 
 FCALC: #scatt.=      4 #anomalous=   0 #special pos.=  1 occupancies=1
 XRAY>end 
 CNSsolve> 
 CNSsolve>show elem ( rmsd ) ( all ) 
 SELRPN:      4 atoms have been selected out of      4
 (      SITE 1    SITE )    2.0000    
 (      SITE 2    SITE )    1.0000    
 (      SITE 3    SITE )    1.0000    
 (      SITE 4    SITE )    1.0000    
 CNSsolve> 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8258392 bytes
           Maximum dynamic memory overhead:          1152 bytes
           Program started at: 16:08:04 on 05-Jul-2018
           Program stopped at: 16:08:04 on 05-Jul-2018
           CPU time used:       0.0083 seconds
          ============================================================
